package.json

```diff
@@ -22,13 +22,16 @@
   ],
   "license": "OGL-UK-3.0",
   "dependencies": {
+    "@azure/ms-rest-nodeauth": "^3.0.6",
+    "@azure/service-bus": "^1.1.10",
```

Comment:
Did you know these two deps needed to be installed along with `ffc-messaging`? I can't see anything in the `README`.

Reply:
They are installed by `ffc-messaging` so will remove from the deps